actions/main/src/waitfordeployment.sh

```diff
@@ -3,14 +3,15 @@ waitfordeployment () {
   eval $(cat env_vars)
   if [[ "${google_zone}" != "" ]] && [[ "${K8_CLUSTER}" != "" ]]; then
     # wait for deployment to happen
-    sleep 5
+    echo "sleep 15 seconds to wait for ui pod to go down after helm deploy..."
```

Comment:
Bit of a deeper fix but is is possible to do something like we do for the perf tests where we look for the deployed commit out of the deployment's metadata?

Reply:
Seems like a good idea to potentially do in the new action.

Reply:
Yes - great point! With my latest changes, we now check for the latest git hash for both the integration and perf tests.

We were using variations of this logic in three places: integration tests, perf tests, and UI end to end tests. I've combined this into one containerized action step. 
-  For integration, we wait until the branch's hash matches the hash running in the integration deployment.
-  For perf, we wait until the git hash corresponding with the latest version on dev is running on perf.
- For all three, we check if the UI pod is running. 

I've also removed the "waitfordeployment" script from "main", and now point all three to the containerized action.

Reply:
Corresponding PRs:
- jade-data-repo: https://github.com/DataBiosphere/jade-data-repo/pull/896
- jade-data-repo-ui: https://github.com/DataBiosphere/jade-data-repo-ui/pull/924